packages/Path-View-Core.package/SClassDiagram.class/instance/extendDiagramAreaTo..st

```diff
@@ -0,0 +1,10 @@
+geometry
+extendDiagramAreaTo: newBounds
+
+	| bounds |
+	bounds := self boundsInWorld.
+	(bounds top > newBounds top) ifTrue: [self dragTopBy: (bounds top - newBounds top)].
```

Comment:
I dont understand why you only extend the diagram when the newBounds are smaller than the previous ones.